azure-iot-device/tests/common/pipeline/test_pipeline_ops_http.py

```diff
@@ -151,7 +78,7 @@ def test_reason(self, cls_type, init_kwargs):
 
 pipeline_ops_test.add_operation_tests(
     test_module=this_module,
-    op_class_under_test=pipeline_ops_http.SetHTTPConnectionArgsOperation,
+    op_class_under_test=pipeline_ops_http.HTTPRequestAndResponseOperation,
```

Comment:
>op_class_under_test=pipeline_ops_http.HTTPRequestAndResponseOperation, [](start = 4, length = 70)

huh.  apparently this attribute isn't that important

Reply:
I mean.... it is important... it's just that it doesn't cause failure. If you pass the wrong one, you test the wrong one. 

---
In reply to: [427577323](https://github.com/Azure/azure-iot-sdk-python/pull/555#discussion_r427577323) [](ancestors = 427577323)